### DISCUSSION TITLE
How to experiment  with @observable/inputs locally in an @observable/framework project?

### DISCUSSION THREAD

**Discussion by Ombuweb** [observablehq/framework#1856] [Q&A] [answered]
So far, I’ve tried the following steps but I’m getting  the error `Error [RollupError]: Could not resolve entry module "@observablehq/inputs"`:

1. Clone the @observable/inputs repo
2. Run `npm i`
3. `npm run prepublishOnly`

Then, in the @observable/framework’s package.json, I add `"@observablehq/inputs": "file:../inputs"` to the dependencies object.

Then in my script, instead of importing the Inputs from `npm:@observablehq/inputs`, I import them from `@observablehq/inputs`

**Comment by gvanhorne** [comment, marked as answer]
Try `npm install ../inputs --install-links`. I think this may be the same issue as was discussed in https://github.com/observablehq/framework/discussions/1606. I think the approach you've described above is creating a symlink which Framework does not follow. Might not hurt to `npm run clean` first either. Hope that helps!

**Reply by Ombuweb** [reply]
That solved it. Thank you very much
